tests/neg-macros/i16582.check

```diff
@@ -0,0 +1,15 @@
+
+-- Error: tests/neg-macros/i16582/Test_2.scala:5:27 --------------------------------------------------------------------
+5 |  val o2 = ownerDoesNotWork(2) // error
+  |           ^^^^^^^^^^^^^^^^^^^
+  |           Exception occurred while executing macro expansion.
+  |           dotty.tools.dotc.core.CyclicReference: Recursive value o2 needs type
+  |
+  |           See full stack trace using -Ydebug
```

Comment:
-Y is supposed to be for internal compiler flags. Ideally we would use -X -or -V which is meant for things that just increase verbosity I think)